libraries/adaptive-expressions/src/builtinFunctions/dataFunc.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ExpressionEvaluator, EvaluateExpressionDelegate } from '../expressionEvaluator';
+import { ReturnType } from '../returnType';
+import { ExpressionType } from '../expressionType';
+import { FunctionUtils } from '../functionUtils';
+import moment from 'moment';
+
+/**
+ * Return the date of a specified timestamp in m/dd/yyyy format.
```

Comment:
>m [](start = 47, length = 1)

mm

Reply:
Currently, the default month type is "M", which means for “May” is "5", and for "Oct" is "10". If we want to change the default format, I could collect all the requirements and change both C#/js sides at the next stage.